src/pro/javacard/gp/AccessControl.java

```diff
@@ -0,0 +1,769 @@
+package pro.javacard.gp;
```

Comment:
If you add a new file, please specify the license (either MIT or LGPL, as you wish, it is published under LGPL but certain parts get re-used under MIT and I usually add fresh code under MIT)